docs/src/01_introduction_and_goals.adoc

```diff
@@ -2,92 +2,46 @@ ifndef::imagesdir[:imagesdir: ../images]
-****
-.Contents
-Short description of the functional requirements, driving forces, extract (or abstract)
-of requirements. Link to (hopefully existing) requirements documents
-(with version number and information where to find it).
-
-.Motivation
-From the point of view of the end users a system is created or modified to
-improve support of a business activity and/or improve the quality.
-
-.Form
-Short textual description, probably in tabular use-case format.
-If requirements documents exist this overview should refer to these documents.
-
-Keep these excerpts as short as possible. Balance readability of this document with potential redundancy w.r.t to requirements documents.
+* The system will provide non-registered users with the option to sign up.
+* The system will provide unidentified users with the option to log in.
+* The system can only be used by registered users.
+* The system will have a game mode with 9 rounds.
+* In each round, the system will generate a question.
+* In each round, the system will generate four answers, with only one being correct.
+* The system will offer registered users access to the number of games they have played.
+* The system will offer registered users access the number of questions they have answered correctly.
+* The system will offer registered users access the number of questions they have answered incorrectly.
+* The system will offer registered users access the time they have spent within the system.
+* The system will offer registered users access the ranking of the game.
+* The system will set a time limit for registered users to respond to each question.
+
+See the complete functional requirements in the xref:#section-annex[Annex] of the documentation.
 
 
-.Further Information
-
-See https://docs.arc42.org/section-1/[Introduction and Goals] in the arc42 documentation.
-
-****
-
 === Quality Goals
-
-[role="arc42help"]
-****
-.Contents
-The top three (max five) quality goals for the architecture whose fulfillment is of highest importance to the major stakeholders. 
-We really mean quality goals for the architecture. Don't confuse them with project goals.
-They are not necessarily identical.
-
-Consider this overview of potential topics (based upon the ISO 25010 standard):
-
-image::01_2_iso-25010-topics-EN.drawio.png["Categories of Quality Requirements"]
-
-.Motivation
-You should know the quality goals of your most important stakeholders, since they will influence fundamental architectural decisions. 
-Make sure to be very concrete about these qualities, avoid buzzwords.
-If you as an architect do not know how the quality of your work will be judged...
-
-.Form
-A table with quality goals and concrete scenarios, ordered by priorities
-****
+[options="header",cols="1,3"]
+|===
+|Goal|Description
+| Functional suitability | The system must fulfill its intended purpose effectively and efficiently, allowing users to register, log in, play the quiz, and access their historical data.
```

Comment:
Same thing here with **_must_**, the verb that is recommended to be used when writing requirements is _**shall**_. Also historical data could be replaced with something more concrete, like "user statistics".